### DISCUSSION TITLE
Error in clone because of the folder name

### DISCUSSION THREAD

**Discussion by ricardomuller90** [community/community#21791] [New to GitHub]
<p>Hi</p>
<p>I try clone a repository to my pc, but return a error saying:</p>
<p><strong>Receiving objects: 100% (23434/23434), 324.09 MiB | 579.00 KiB / s, done.</strong><br>
<strong>Resolving deltas: 100% (4964/4964), done.</strong><br>
<strong>error: invalid path 'C: \ pharos \ storage \ logs / laravel.log’</strong><br>
<strong>fatal: can not create directory at ‘pharos / public / C: \ pharos \ storage \ logs’: Invalid argument</strong><br>
<strong>warning: Clone succeeded, but checkout failed.</strong><br>
<strong>You can check what was checked out with 'git status’</strong><br>
<strong>and retry the checkout with 'git checkout -f HEAD’</strong></p>
<p>Making a search, i think the error is because of the folder name which has the character ‘’.</p>
<p>I am right? If yes, how do I rename the folder name directly in git hub?</p>

**Comment by Wabri** [comment]
<p>Hi <a class="mention" href="/ricardomuller90">@ricardomuller90</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>Could you please post the command run? or the desktop app you used?</p>
<p>Something strange indeed, it’s like a windows unix combo mix.</p>
<p>Look at <strong>pharos/public/C:\storage\logs</strong> , there is two different approach:</p>
<ul>
<li>unix format for <strong>pharos/public/</strong>
</li>
<li>windows format for <strong>C:\storage\logs</strong>
</li>
</ul>
<p>Do you use a linux console (like ubuntu) inside windows?</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Comment by ricardomuller90** [comment]
<p>Hi!</p>
<p>Yes!</p>
<p>to clone i used the command: git clone </p>
<p>i’m using the gitbash terminal in windows.</p>

**Comment by ricardomuller90** [comment, marked as answer]
<p>I solved excluding the file, so the folder disappear and I could make the clone have successful. thanks</p>

**Comment by raheemuddin786** [comment]
<p>how did you excluded, can you please elaborate the solution with an example. </p>
<p>Thanks in advance.</p>
